### PR TITLE
Import Statement Use Deprecated @algorand-builder/algob

### LINKED ISSUE #2: Import Statement Use Deprecated @algorand-builder/algob
`const { SignType, TransactionType, executeTransaction, balanceOf } = require('@algorand-builder/algob');`

In this line, https://github.com/scale-it/algorand-builder-tutorial/blob/part-1/scripts/0-gold.js#L1 please replace it with the new library

### PR DESCRIPTION
**Pull request by rmeena840**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,8 @@
   "main": "index.js",
   "license": " MPL-2.0",
   "dependencies": {
-    "algob": ">=0.3.0"
+    "@algo-builder/algob": "^5.0.1",
+    "@algo-builder/web": "^5.0.1"
   },
   "scripts": {
     "algob": "algob"
```
Comment (ratik21) [inline package.json]: we should remove this, if you're using direct dependency. Otherwise add a node for `yarn link`

File: scripts/0-gold.js [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
     toAccountAddr: receiverAddr,
     amountMicroAlgos: amount,
     payFlags: payFlags
   };
 };
 
-
 // This is an entry function in our script (a default, exported function)
-async function run (runtimeEnv, deployer) {
+async function run(runtime, deployer) {
```
Comment (ratik21) [inline scripts/0-gold.js:17]: does this need to be changed?

**Review by vuvoth — COMMENTED** [review]
Method `algob.balanceOf()` now return value, so you should use balaceOf with console.log e.g `console.log(await balaceOf())`.

File: scripts/0-gold.js (hunk 2 above)
Comment (rmeena840) [reply scripts/0-gold.js:17]: Not needed but in algo-builder repo, we are using the same variable name. So thought of applying same here.

**Comment by rmeena840** [comment]
@vuvoth Changes done!!

File: package.json (hunk 1 above)
Comment (rmeena840) [reply package.json]: Thanks. Changes done.

File: scripts/transfer/gold-to-john.js [hunk 3]
```diff
@@ -13,15 +14,15 @@ async function run (runtimeEnv, deployer) {
...
+    sign: types.SignType.SecretKey,
     fromAccount: goldOwner,
     toAccountAddr: john.addr,
     amount: 1,
     assetID: goldAsset.assetIndex,
     payFlags: {}
-  });
+  }]);
 
   await balanceOf(deployer, john.addr, goldAsset.assetIndex);
```
Comment (vuvoth) [inline scripts/transfer/gold-to-john.js]: You forgot update this line 😅

**Review by vuvoth — APPROVED** [review]

File: scripts/transfer/gold-to-john.js (hunk 3 above)
Comment (rmeena840) [reply scripts/transfer/gold-to-john.js]: It's there. You have commented on the old change.

Comment (vuvoth) [reply scripts/transfer/gold-to-john.js]: I see you only update `scripts/0-gold.js`.

Comment (rmeena840) [reply scripts/transfer/gold-to-john.js]: We discussed

**Pull request merged**